PhpAmqpLib/Connection/AbstractConnection.php

```diff
@@ -559,26 +559,25 @@ protected function wait_channel($channel_id, $timeout = 0)
                 }
                 continue;
 
-            } else {
```

Comment:
AFAIK the `continue` will make the `while` loop above to jump to the next iteration, so this is not related to the `else` below